util/app-config/src/cli.rs

```diff
@@ -245,7 +246,13 @@ fn import() -> App<'static, 'static> {
 }
 
 fn migrate() -> App<'static, 'static> {
-    SubCommand::with_name(CMD_MIGRATE).about("Runs ckb migration")
+    SubCommand::with_name(CMD_MIGRATE)
+        .about("Runs ckb migration")
+        .arg(
+            Arg::with_name(ARG_MIGRATE_CHECK)
+                .long(ARG_MIGRATE_CHECK)
+                .help("Perform database version check without migrating"),
```

Comment:
Please add detailed document about the exist code: what does exit code 0 means.

Reply:
done. :haircut_man: